src/abap/types/class_definition.ts

```diff
@@ -82,6 +82,10 @@ export class ClassDefinition extends Identifier {
     return this.node.findFirstStatement(Statements.ClassDefinition)!.concatTokens().toUpperCase().includes(" ABSTRACT");
   }
 
+  public isRedefinition(): boolean {
```

Comment:
the idea is correct, but the redefinition is in the scope of the method, not the class itself.
check the class `MethodDefinition`

Reply:
Thanks, you are right, now I changed test cases and determine Redefinition on method level.